spec/unit/provider/user/hpux_spec.rb

```diff
@@ -49,4 +50,23 @@
       provider.password.must == "foopassword"
     end
   end
+
+  context "Check for trusted computing" do
+  before :each do
+    provider.stubs(:command).with(:modify).returns '/usr/sam/lbin/usermod.sam'
+  end
+
+  it "Should add modprpw to modifycmd if Trusted System" do
+    resource.stubs(:allowdupe?).returns true
+    provider.stubs(:exec_getprpw).with('root','-m uid').returns('uid=0')
+    provider.expects(:execute).with { |args| args.include?("/usr/lbin/modprpw") }
```

Comment:
I'd be more comfortable with this asserting the whole command that will be executed.